src/screens/Tag.js

```diff
@@ -0,0 +1,230 @@
+} from '../utils/genericPropExamples';
+import {
+  ComponentDoc,
+  Properties,
+  Property,
+  PropertyValue,
+  Description,
+  Example,
+  ThemeDoc,
+  PropOptions,
+} from '../components/Doc';
+
+export default () => (
+  <Page>
+    <ComponentDoc
+      name="Tag"
+      availableAt={[
+        {
+          url: 'https://storybook.grommet.io/?selectedKind=Type-Tag&full=0&stories=1&panelRight=0',
+          badge:
+            'https://cdn-images-1.medium.com/fit/c/120/120/1*TD1P0HtIH9zF0UEH28zYtw.png',
+          label: 'Storybook',
+        },
+        {
+          url: 'https://codesandbox.io/s/github/grommet/grommet-sandbox?initialpath=/tag&module=%2Fsrc%2FTag.js',
+          badge: 'https://codesandbox.io/static/img/play-codesandbox.svg',
+          label: 'CodeSandbox',
+        },
+        {
+          url: 'https://github.com/grommet/grommet/tree/master/src/js/components/Tag',
+          label: 'Github',
+        },
+      ]}
+      description="A component to represent a textual tag"
+      intrinsicElement="div"
+      code={`<Tag name="name" value="value" />`}
+    >
+      <Properties>
+        <Property name="a11yTitle">
+          <Description>
+            Custom label to be used by screen readers. When provided, an
+            aria-label will be added to the element.
+          </Description>
+          <GenericA11yTitle />
+        </Property>
+        
+        <Property name="alignSelf">
+          <Description>
+            How to align along the cross axis when contained in a Box or along
+            the column axis when contained in a Grid.
+          </Description>
+          <GenericAlignSelf />
+        </Property>
+        
+        <Property name="as">
+          <Description>
+            The DOM tag or react component to use for the element.
+          </Description>
+          <GenericAs />
+        </Property>
+
+        <Property name="gridArea">
+          <Description>
+            The name of the area to place this inside a parent Grid.
+          </Description>
+          <GenericGridArea />
+        </Property>
+
+        <Property name="name">
+          <Description>Name text to place in the tag.</Description>
+          <PropertyValue type="string">
+            <Example>"name"</Example>
+          </PropertyValue>
+        </Property>
+        <Property name="onClick">
+          <Description>
+            Click handler. If this property is specified the tag will be a
+            button.  This should not be used in conjunction with 'onRemove'.
+          </Description>
+          <PropertyValue type="function">
+            <Example>{`() => {}`}</Example>
+          </PropertyValue>
+        </Property>
+        
+        <Property name="onRemove">
+          <Description>
+            If specified, the tag will include a remove button. This function
+            will be called when the remove button is activated. This should not
+            be used in conjunction with 'onClick'.
```

Comment:
```suggestion
```

Reply:
Why remove it here?

Reply:
My comment above adjusted the whole sentence -- since this was broken onto its own line, it's now repeated with my suggestion above. Only commit this if we accept my above suggestion.